api-conta-bancaria/src/main/java/com/apicontabancaria/domain/model/Conta.java

```diff
@@ -0,0 +1,101 @@
+package com.apicontabancaria.domain.model;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+
+@Entity
+public class Conta {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	private Double saldo;
+
+	private String tipoConta;
+
+	@Enumerated(EnumType.STRING)
+	private StatusConta statusConta;
+
+	@OneToOne
+	private Cliente cliente;
+
+	public Conta() {
+
+	}
+
+	public Conta(Conta conta) {
```

Comment:
public Conta(Long id, Double saldo, String tipoConta) {
                this.id = id;
		this.saldo = saldo;
		this.tipoConta = tipoConta;
	}